### DISCUSSION TITLE
Can´t install ?

### DISCUSSION THREAD

**Discussion by mendes588** [punxaphil/custom-sonos-card#105] [Q&A] [answered]
Good afternoon,

I have no idea why, but it seems like the Sonos Card doesn't show up.  It keeps telling me, the custom element doesn't exist. Regardless if I use YAML.

I added the card via HACS. Any idea what the problem is ?  I went straight for the 5.17.- Restarted HA. Checked resource links..

Do I need any other Integration ? How would I try to install it manually? I would need to compile the file ? Sorry, but I feel like an amateur.
Thank you in advance,

**Comment by mendes588** [comment, marked as answer]
I got it to work magically. I dunno why, but the resource location is wrong via HACS somehow. Had to manually update it and wait a bit for it to load.

Thanks. Maybe someone can have a look into it.

**Reply by punxaphil** [reply]
Great that you got it working. If many people start reporting this, it should probably be investigated.

**Comment by pasi64** [comment]
I've the same problem to integrate it in hacs.
![integrate](https://github.com/johanfrick/custom-sonos-card/assets/134091632/1208a6c3-b276-4d2a-b800-b1995f17bb28)

I've placed the github-files in the www-folder with subfolder sonos.
No luck with integration ...

After that i tried to generate a custom card with error ... 

![customcard](https://github.com/johanfrick/custom-sonos-card/assets/134091632/ebda57ea-c6f7-4fe5-a9ea-b63913a5084d)

Please can you give me more instructions to integrate it

**Reply by punxaphil** [reply]
I you are using the latest beta version of the card, then you must use `custom:sonos-card` not `custom:sonos-player`.
